samples/nativeSample/src/main/java/com/inmobi/nativead/sample/singlestrand/SingleNativeAdFragment.java

```diff
@@ -83,26 +84,30 @@ public void onConfigurationChanged(Configuration newConfig) {
     }
 
     private void createStrands() {
-        mInMobiNative = new InMobiNative(getActivity(), PlacementId.YOUR_PLACEMENT_ID_HERE, new StrandAdListener());
-        mInMobiNative.setVideoEventListener(new VideoEventListener() {
-            @Override
-            public void onVideoCompleted(InMobiNative inMobiNative) {
-                super.onVideoCompleted(inMobiNative);
-                Log.d(TAG, "Video completed");
-            }
-
-            @Override
-            public void onVideoSkipped(InMobiNative inMobiNative) {
-                super.onVideoSkipped(inMobiNative);
-                Log.d(TAG, "Video skipped");
-            }
-
-            @Override
-            public void onAudioStateChanged(InMobiNative inMobiNative, boolean b) {
-                super.onAudioStateChanged(inMobiNative, b);
-                Log.d(TAG, "Audio state changed");
-            }
-        });
+        try {
+            mInMobiNative = new InMobiNative(getActivity(), PlacementId.YOUR_PLACEMENT_ID_HERE, new StrandAdListener());
+            mInMobiNative.setVideoEventListener(new VideoEventListener() {
+                @Override
+                public void onVideoCompleted(InMobiNative inMobiNative) {
+                    super.onVideoCompleted(inMobiNative);
+                    Log.d(TAG, "Video completed");
+                }
+
+                @Override
+                public void onVideoSkipped(InMobiNative inMobiNative) {
+                    super.onVideoSkipped(inMobiNative);
+                    Log.d(TAG, "Video skipped");
+                }
+
+                @Override
+                public void onAudioStateChanged(InMobiNative inMobiNative, boolean b) {
+                    super.onAudioStateChanged(inMobiNative, b);
+                    Log.d(TAG, "Audio state changed");
+                }
+            });
+        } catch (SdkNotInitializedException e) {
```

Comment:
`try/catch` block is not necessary for `SdkNotInitializedException` as the intention is to throw exception only for wrong integration. Publishers following the guidelines need not even handle this exception as it will never occur.